juniper_official/Kernel/check-stuck-ifstate-clients.rule

```diff
@@ -0,0 +1,48 @@
+/*
+ * This rule checks if there are any stuck ifstate clients present in the system.
+ *
+ * A stuck Ifstate client can cause resource release issues and eventually lead to
+ * Kernel to veto incoming routing socket messages from applications, which further
+ * can result in traffic loss. Though very rare, if this happens, it is a serious
+ * condition and should be mitigated asap by restarting the client. See readme
+ * in healthbot rules folder for details.
+ */
+iceberg {
+    topic kernel.ifstate {
+        rule check-stuck-ifstate-clients {
+            sensor ifstate-clients {
+                open-config {
+                    sensor-name /junos/kernel-ifstate;
+                    frequency 1m;
+                }
+            }
+            field stuck-clients {
+                sensor ifstate-clients {
+                    path /junos/kernel-ifstate/stuck-clients-cnt;
+                }
+                type integer;
+            }
+            trigger report-ifstate-clients-health {
```

Comment:
1) Provide trigger description and synopsis detailing sensor info.
2) Trigger frequency is must as default frequency is 10s and trigger gets evaluated once in 10s. but here sensor frequecy is 10 mins hence trigger frequency can be greaterthan equal to 10 mins.
e.g. synopsis "Routing engine GRES kpi";
       description "sets health based on Routing engine CPU utilization for 15 minutes";

Reply:
Addressed.